### DISCUSSION TITLE
pm2重启原因

### DISCUSSION THREAD

**Discussion by left0ver** [takayama-lily/oicq#356] [Q&A]
### Bug Description (最好截图说明，你的文字描述可能无法准确传达信息)

我把代码放到pm2上管理，想每天定时某个时间在群里发消息，因此我设置了一个定时器，每隔一天定时发送消息，但是我发现每次早上8点的时候和下午4点的时候，pm2就会重启，就会导致重新执行一遍代码。但是我另外有一个网页的项目就不会重启，有没有人知道这其中的原因

### Log Details (控制台日志，请提供包含前后一段时间内的)

![A%5XH9T5$EO5$3T6H`@BA`S](https://user-images.githubusercontent.com/89240290/159711278-88ad6fb8-2380-418e-a523-ce3442fee220.png)
![1S @~}T_B{J(I%MFXI2XH@S](https://user-images.githubusercontent.com/89240290/159711475-6659971b-e7f2-4e4f-ab18-925406401c64.png)

**Comment by dogxii** [comment]
定时器发生报错？
